deployments/package.json

```diff
@@ -1,16 +1,12 @@
 {
   "name": "@nexusmutual/deployments",
-  "version": "2.1.1",
+  "version": "2.2.0",
   "description": "Nexus Mutual deployed contract addresses and abis",
-  "exports": {
-    ".": "./dist/index.js",
-    "./abis": "./dist/abis.js",
-    "./abis/*": "./dist/abis/*.js",
-    "./addresses": "./dist/addresses.js"
-  },
+  "typings": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
   "files": [
-    "dist",
-    "README.md"
```

Comment:
Removed `README.md` because it is [included by default](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files).